### PR TITLE
changed: make standard adaptive formulations output first mesh to eps

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

File: src/ASM/LR/ASMLRSpline.C [hunk 1]
```diff
@@ -143,15 +143,40 @@ bool ASMunstruct::refine (const LR::RefineData& prm,
 {
   PROFILE2("ASMunstruct::refine()");
 
+  auto&& storeMesh = [this, fName]()
```
Comment (kmokstad) [inline src/ASM/LR/ASMLRSpline.C]: Lambda functions is a nice concept, but in this case I think a protected member function would be a more natural choice?

Comment (akva2) [reply src/ASM/LR/ASMLRSpline.C]: Probably, i just did the same as was done for mixed.

**Comment by akva2** [comment]
changed to priv member.

**Comment by akva2** [comment]
jenkins build this please

File: src/ASM/ASMunstruct.h [hunk 2]
```diff
@@ -227,6 +227,11 @@ class ASMunstruct : public ASMbase
 
   static int gEl;  //!< Global element counter
   static int gNod; //!< Global node counter
+
+private:
+  //! \brief Store basis to eps files.
+  //! \param fName Prefix for file names
```
Comment (kmokstad) [inline src/ASM/ASMunstruct.h]: `\param[in]`

File: src/ASM/LR/ASMLRSpline.C [hunk 3]
```diff
@@ -143,15 +143,19 @@ bool ASMunstruct::refine (const LR::RefineData& prm,
 {
   PROFILE2("ASMunstruct::refine()");
 
+
```
Comment (kmokstad) [inline src/ASM/LR/ASMLRSpline.C]: Remove extra blank line

**Pull request merged**